docs/OESDLBugBar.md

```diff
@@ -0,0 +1,84 @@
+# Open Enclave SDL Bug Bar
+
+The information listed in this “bug bar” is intended to help Open Enclave SDK developers to triage bugs and determine the bug severity in terms of security. 
+
+This bug bar and the ratings are derived based of Microsoft SDL Bug bar to apply for the enclaves, specifically to the enclaves created using Intel SGX technology. It is highly recommended to fix all known bugs with critical, important, or moderate severity prior to release. 
```

Comment:
From this document, it is unclear what "Microsoft SDL Bug bar" is. Could you do one of the following:

1. Link to a documnet explaining the source of this concept
2. Try to clear the document of MS-specific language and instead describe the goals in generic terms. I prefer this route, and if we go this route, please add a quick definition to the beginning about what "bug bar" means

Reply:
I will add a hyperlink to the standard Microsoft SDL Bug Bar, I hope that should be sufficient.